orc8r/gateway/python/magma/magmad/bootstrap_manager.py

```diff
@@ -411,3 +414,28 @@ def _ecdsa_sha256_response(self, challenge):
         r_bytes = r_int.to_bytes((r_int.bit_length() + 7) // 8, 'big')
         s_bytes = s_int.to_bytes((s_int.bit_length() + 7) // 8, 'big')
         return r_bytes, s_bytes
+
+    async def _request_vpn_cert(self):
+        try:
+            chan = ServiceRegistry.get_bootstrap_rpc_channel()
+        except ValueError as exp:
+            logging.error('Failed to get rpc channel: %s', exp)
+            return
+
+        client = BootstrapperStub(chan)
+        try:
+            result = await grpc_async_wrapper(
+                client.RequestVPNCert.future(Void()),
```

Comment:
Does this resolve to anything? I don't see the diff for the RPC method on this PR